standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1825,6 +1825,24 @@ public enum ConfVars {
         "hive.metastore.properties.servlet.auth", "jwt",
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    PROPERTIES_NS("hive.metastore.properties.namespaces",
+        "hive.metastore.properties.namespaces", "",
+        "Property-maps namespace map; a JSON map where keys are namespace names, values namespace" +
+            "class and/or static factory method"),
+    CATALOG_SERVLET_PATH("hive.metastore.catalog.servlet.path",
+        "hive.metastore.catalog.servlet.path", "icecli",
+        "Catalog servlet path component of URL endpoint."
+    ),
+    CATALOG_SERVLET_PORT("hive.metastore.catalog.servlet.port",
+        "hive.metastore.catalog.servlet.port", -1,
+        "Catalog servlet server port. Negative value disables the servlet," +
+            " 0 will let the system determine the catalog server port," +
+            " positive value will be used as-is."
+    ),
+    CATALOG_SERVLET_AUTH("hive.metastore.catalog.servlet.auth",
```

Comment:
Please use more appropriate name here. I could not find out neither from the name, nor from the description this property is for Iceberg Rest Server